libethereum/BlockQueue.cpp

```diff
@@ -179,6 +167,21 @@ void BlockQueue::verifierBody()
 	}
 }
 
+void BlockQueue::drainVerified()
+{
+	while (m_verifying.size() && !m_verifying.front().blockData.empty())
```

Comment:
language sanitizers generally prefer the always-fastest `!m_verifying.empty()` over `m_verifying.size()`.